src/QMCDrivers/WalkerProperties.h

```diff
@@ -0,0 +1,49 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2019 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Lab
+//
+// File created by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Lab
+//////////////////////////////////////////////////////////////////////////////////////
+#ifndef QMCPLUSPLUS_WALKERPROPERTIES_H
+#define QMCPLUSPLUS_WALKERPROPERTIES_H
+
+#include <cstdint>
+#include "config.h"
+
+namespace qmcplusplus
+{
+struct WalkerProperties
+{
+  /** an enum denoting index of physical properties
+ *
+ * \todo: this enum and the handling of "Properties" through the class hierarchy
+ * is hot garbage. Replace, for now we're just making it safe.
+ *
+ * LOCALPOTENTIAL being defined last of the Walker "Properties" is special there
+ * is likely to be some code lurking that depends on it. Keep until this mess is removed.
+
+ * Believe it or not,you need to modify ParticleSet::initPropertyList to match the walker enum if you add to it.
+ * _Don't_, actually work on fixing this.
+ */
+  enum Indexes : int16_t
```

Comment:
Why not
```
enum class WalkerProperties
{
}
```

Reply:
Requires a `static_cast<int>` for every use as an index. Since other "users" of Walker::Properties use arbitrary int indexes into it for >= WP::NUMPROPERTIES we can't define a helper function that forces accessor to use type WalkerProperties and do the cast inside the accessor.

Once we retire MCWC with the legacy drivers. I'd like to refactor walker and the use of PooledMemory in it. This could allow the Properties issue to be handled in a different manner.

Reply:
Maybe this could work? Anyway not critical.
```
enum class WalkerProperties : int
{
}
```